diff --git a/lib/cjs/launch.js b/lib/cjs/launch.js
index 20ed82ec18dda198fe69a7bc64097694e74361d5..55522116ca78562314d1be6a3b7566e3cf2ef53a 100644
--- a/lib/cjs/launch.js
+++ b/lib/cjs/launch.js
@@ -250,6 +250,8 @@ class Process {
         return this.#browserProcessExiting;
     }
     kill() {
+        Error.stackTraceLimit = Infinity;
+        debugLaunch(new Error().stack);
         debugLaunch(`Trying to kill ${this.#browserProcess.pid}`);
         // If the process failed to launch (for example if the browser executable path
         // is invalid), then the process does not get a pid assigned. A call to
diff --git a/lib/esm/install.d.ts b/lib/esm/install.d.ts
index 3bcad4c91764081ac431d73c7448978af11e942e..64c438499dcab54d541e70d9077b96b0c6c09509 100644
--- a/lib/esm/install.d.ts
+++ b/lib/esm/install.d.ts
@@ -5,6 +5,9 @@
  */
 import { Browser, BrowserPlatform } from './browser-data/browser-data.js';
 import { InstalledBrowser } from './Cache.js';
+
+export declare function getDownloadUrl(browser: Browser, platform: BrowserPlatform, buildId: string, baseUrl?: string): URL;
+
 /**
  * @public
  */
diff --git a/lib/esm/install.js b/lib/esm/install.js
index 25c2df862da874c27d6bae1cc25d999fb47b81db..2a3616218c372b5714aa60aea979c316d4deead6 100644
--- a/lib/esm/install.js
+++ b/lib/esm/install.js
@@ -241,7 +241,7 @@ export async function canDownload(options) {
     }
     return await headHttpRequest(getDownloadUrl(options.browser, options.platform, options.buildId, options.baseUrl));
 }
-function getDownloadUrl(browser, platform, buildId, baseUrl) {
+export function getDownloadUrl(browser, platform, buildId, baseUrl) {
     return new URL(downloadUrls[browser](platform, buildId, baseUrl));
 }
 //# sourceMappingURL=install.js.map
\ No newline at end of file
diff --git a/lib/esm/launch.js b/lib/esm/launch.js
index 9e75b536a883a62934461284263d2a71df2f06b4..a83b342f8edb5a270fb55e4c98da4231600690d3 100644
--- a/lib/esm/launch.js
+++ b/lib/esm/launch.js
@@ -239,6 +239,7 @@ export class Process {
         return this.#browserProcessExiting;
     }
     kill() {
+        debugLaunch(new Error().stack);
         debugLaunch(`Trying to kill ${this.#browserProcess.pid}`);
         // If the process failed to launch (for example if the browser executable path
         // is invalid), then the process does not get a pid assigned. A call to
diff --git a/package.json b/package.json
index 75b5e681eb5be626c6b2faf274d4ff1c9ea9901d..6cd64ceabb94f39292eb74f7796800ef6229b6a2 100644
--- a/package.json
+++ b/package.json
@@ -12,10 +12,13 @@
   "type": "commonjs",
   "bin": "lib/cjs/main-cli.js",
   "main": "./lib/cjs/main.js",
-  "exports": {
+  "exports": {".": {
     "import": "./lib/esm/main.js",
     "require": "./lib/cjs/main.js"
-  },
+  }, "./*": {
+    "types": "./lib/esm/*.d.ts",
+    "import": "./lib/esm/*.js"
+  }},
   "wireit": {
     "build": {
       "command": "tsc -b && node --experimental-strip-types ../../tools/chmod.ts 755 lib/cjs/main-cli.js lib/esm/main-cli.js",
