diff --git a/src/importOrRequireConfig.js b/src/importOrRequireConfig.js
index 2acb498aca14d83ee1dcd941b80b231e40929838..b69ec51ef2e36d345e4b9f138afc6796573cea83 100644
--- a/src/importOrRequireConfig.js
+++ b/src/importOrRequireConfig.js
@@ -17,7 +17,6 @@ function importConfig(configPath) {
  * @param {string} basedir
  */
 async function importOrRequireConfig(configPath, basedir) {
-  const packageType = await getPackageType(basedir);
   const ext = path.extname(configPath);
 
   switch (ext) {
@@ -26,6 +25,7 @@ async function importOrRequireConfig(configPath, basedir) {
     case '.cjs':
       return requireConfig(configPath);
     default:
+      const packageType = await getPackageType(basedir);
       return packageType === 'module' ? importConfig(configPath) : requireConfig(configPath);
   }
 }
